.github/workflows/ci.yml

```diff
@@ -113,3 +113,41 @@ jobs:
         run: sudo xcode-select --switch /Applications/Xcode_13.4.1.app/Contents/Developer
       - name: Build and Test Framework
         run: xcrun swift test -c release -Xswiftc -enable-testing
+  spm-14:
+    name: Build Xcode 14
+    runs-on: macOS-12
+    strategy:
+      matrix:
+        platforms: [
+          'iOS_16,watchOS_9',
+          'macOS_12,tvOS_16',
```

Comment:
Note that we're testing `macOS_12` here again. We don't _need_ to do this but given that we're running with a different SDK I figure why not.